rkt/pubkey/pubkey.go

```diff
@@ -101,24 +96,22 @@ func (m *Manager) AddKeys(pkls []string, prefix string, accept AcceptOption, ove
 		}
 		defer pk.Close()
 
-		exists, err := m.Ks.TrustedKeyPrefixExists(prefix, pk)
-		if err != nil {
-			return errwrap.Wrap(fmt.Errorf("error reading the key %s", pkl), err)
-		}
 		err = displayKey(prefix, pkl, pk)
 		if err != nil {
 			return errwrap.Wrap(fmt.Errorf("error displaying the key %s", pkl), err)
 		}
-		if exists && override == OverrideDeny {
-			log.Printf("key %q already in the keystore", pkl)
-			continue
-		}
 
 		if m.TrustKeysFromHTTPS && u.Scheme == "https" {
 			accept = AcceptForce
 		}
 
 		if accept == AcceptAsk {
+			if !terminal.IsTerminal(int(os.Stdin.Fd())) || !terminal.IsTerminal(int(os.Stderr.Fd())) {
+				stdout.Printf("To trust the key for %q, do one of the following:", prefix)
```

Comment:
I was expecting to see a `stderr` in addition to a `stdout`. What's the difference between `stdout` and `log`?